h2o-algos/src/main/java/hex/deeplearning/DeepLearning.java

```diff
@@ -326,6 +338,13 @@ public final void buildModel() {
         Scope.exit(keep.toArray(new Key[keep.size()]));
       }
     }
+    
+    void initEffectiveParam(DeepLearningModel deepLearningModel) {
```

Comment:
why not in model, here the orignal model. initEffectiveParam is not even ever called

Reply:
you're right, this whole initEffectiveParam was to be deleted and and original model.initEffectiveParam was to be called in computeEffectiveParameters() on line 224. Thanks!

Reply:
fixed